pyaerocom/conftest.py

```diff
@@ -156,6 +161,12 @@ def loaded_nasa_ames_example():
     #fp = TESTDATADIR.joinpath(TEST_PATHS['nasa_ames_sc550aer'])
     return EbasNasaAmesFile(NASA_AMES_FILEPATHS['scatc_jfj'])
 
+@pytest.fixture(scope='session')
+def tempdir(tmpdir_factory):
+    """Temporary directory for dumping data shared between tests"""
+    tmpdir= tmpdir_factory.mktemp('data')
+    return tmpdir
```

Comment:
Is this meant to be persistent between different test files?
If not I think this should have scope='module'.

Reply:
Yes it is (I think the idea was to reload files created in a different module, such as colocated data objects).